### ISSUE TITLE
view command shouldn't require book name

### ISSUE THREAD

**Issue by sungwoncho** [dnote/dnote#161]
View command takes book name as the first argument but that requirement is redundant because the command takes a unique note id as the third argument. For instance,

    dnote v myBook 123

might well be

    dnote v 123

because `123` is a unique identifier for a note.

The intention for taking book name as an argument was to save users from typing `dnote view` in the following scenario:

```
dnote v myBook
# ...result is shown...
# user presses up key, and types the note id
dnote v myBook 123
```

But the scenario does not happen as often. Also it is inconvenient to have to type book name after using the `find` command.

* this change means that book name can't be integer. Need to migrate both client and server side for book names that violate this condition.

---

plan:

* support `view` command with a single argument
* deprecate two arguments
* release 0.6.0

**Comment by sungwoncho** [comment]
In progress in [simple-view](https://github.com/dnote/cli/tree/simple-view) branch. Not a huge priority.

**Comment by sungwoncho** [comment]
Fixed by #184
